### ISSUE TITLE
TypeError: XLSX.utils.json_to_sheet is not a function

### ISSUE THREAD

**Issue by rajatady** [SheetJS/js-xlsx#811]
Including the CDN version of the library is giving this error. The version I am including is **0.11.3**. The same version works fine when installed using bower during development. The screenshots of the XLSX.utils object referenced using 1. **CDN** and 2. **Bower** are attached.
![screen shot 2017-09-18 at 7 36 08 pm](https://user-images.githubusercontent.com/11324121/30545952-dc99df6c-9ca8-11e7-9616-affbb5ab2024.png)
![screen shot 2017-09-18 at 7 36 26 pm](https://user-images.githubusercontent.com/11324121/30545953-dca183de-9ca8-11e7-818b-3c8373d34153.png)

**Comment by reviewher** [comment]
Which CDN / script are you using?

CDNJS: https://jsfiddle.net/7jf2Ldpu/ uses https://cdnjs.cloudflare.com/ajax/libs/xlsx/0.11.3/xlsx.full.min.js

UNPKG: https://jsfiddle.net/7jf2Ldpu/2/ uses https://unpkg.com/xlsx@0.11.3/dist/xlsx.full.min.js

**Comment by rajatady** [comment]
I am using cdnjs.
https://cdnjs.cloudflare.com/ajax/libs/xlsx/0.11.3/xlsx.full.min.js

**Comment by reviewher** [comment]
The first fiddle https://jsfiddle.net/7jf2Ldpu/ uses that same script and dumps all of the available items in `XLSX.utils`.  I see `json_to_sheet` in the list in the lower-right pane.  Can you open that page and check if the output shows it?

<img width="896" alt="screen shot 2017-09-18 at 12 44 29" src="https://user-images.githubusercontent.com/24845478/30553422-28966d8e-9c6f-11e7-914e-d0b2e5507aaf.png">

**Comment by rajatady** [comment]
Indeed. The JsFiddle has the json_to_sheet function on the XLSX.utils object. But our prod website is not able to access the function. I checked it again, its not pulling up the sheet_to_json function. Still giving the same error.

**Comment by reviewher** [comment]
That looks like an error with your build configuration -- is it possible you are accidentally pulling in another version elsewhere? Just before the line with `XLSX.utils.json_to_sheet`, try checking the version with `console.log(XLSX.version)`.

**Comment by animishas** [comment]
were you able to resolve this issue? @rajatady I was having same issue and I have the same version 0.11.3.
